### DISCUSSION TITLE
Enhancement: New UI and Information

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1678] [Ideas]
I have been thinking this for a while. The current freeplay and storymode menu are not final i know that but i have a suggestion for that. 

Part 1 : New UI

For story mode and freeplay merge them into one tab, make a new menu "Map" or "Storymode" or similar and that displays a visual map of the FnF world like the image below 
(Please keep in mind all drawings/diagrams are crude)
![image](https://user-images.githubusercontent.com/85327029/126891712-b335a315-d79b-4290-bd6a-b609db355587.png)
For the "freeplay" aspect of it when we click on a week it opens a new menu for us to navigate (probably inspired by the old menu)
![image](https://user-images.githubusercontent.com/85327029/126891910-335b91c3-56f4-4aa7-9ff4-be70efbae2be.png)

Part 2 : New Information
This is more of a smaller part of the game than a real suggestion but id still like to voice my thoughts. As shown in the previous image it gives the description of Daddy Dearest. We could have them as a small popup for weeks when hovering over them
![image](https://user-images.githubusercontent.com/85327029/126892100-d95b38c1-3117-49d0-aa2e-f42fe05ad113.png)
Another small information based feature that could be implemented is descriptions on the following:
* Difficulty 
* Charecters (Probably through an archive of sorts like in pokemon)
* Locations (I dont know how this could be implemented, But again an archive of sorts would help)

Difficulty Information
![image](https://user-images.githubusercontent.com/85327029/126892224-f1bc7305-b238-4a9c-b268-fda7819b1ee0.png)
![image](https://user-images.githubusercontent.com/85327029/126892257-4be65e20-7551-4908-a8b3-eccc8917a025.png)
![image](https://user-images.githubusercontent.com/85327029/126892284-9c4b18a8-520d-4dae-9efb-2ad5121dfe3d.png)

Charecter Information
![image](https://user-images.githubusercontent.com/85327029/126893250-0e1c41e1-0440-4d45-a7bd-d5427a1514fb.png)

Probably with less information and better formatting
![image](https://user-images.githubusercontent.com/85327029/126893687-52be8807-a75e-409f-a6a7-4c11ce0db0ad.png)

Locations
This is more of a stupid idea but i just wanted to be open with myself
![image](https://user-images.githubusercontent.com/85327029/126893861-15abb4e4-dc15-4b44-9d4c-41facf69e626.png)

It is okay if this does not get added to the game. I was happy you read through this idea and paid attention
Have a good day

**Comment by Windows1Door2Daniel** [comment]
Looks nice

**Comment by ghost** [comment]
really? i made them as extreme crude placeholders for the devs to build on

**Comment by libauror-a** [comment]
Looks cool, might steal your idea and inplement this ui at some point >:)

**Comment by ghost** [comment]
@ThePlank that is the main reason i made this XD
feel free to copy paste my ideas into the game i dont mind at all

**Comment by ghost** [comment]
one thing: norma;l has most but not all, hard has all

**Comment by ghost** [comment]
why not

**Comment by inue-kang** [comment]
Not bad, you should check out FNF Mic'd Up Engine tho, it has a pretty cool UI.

**Comment by ghost** [comment]
HES A GENIUS I HOPE THEY ADD THAT

**Comment by ghost** [comment]
Yeah i hope one of the contributors see it and ask ninjamuffin for its addition
